app/networkController.js

```diff
@@ -1,6 +1,6 @@
 // This file is used to make API requests to the server.
 
-const BASE_URL = 'http://192.168.1.21:4444'; // Assuming you are using an Android emulator. Use 'http://localhost:4444' for iOS.
+const BASE_URL = 'http://10.8.1.245:4444'; // Assuming you are using an Android emulator. Use 'http://localhost:4444' for iOS.

```

Comment:
We should stop committing this change in the future

Reply:
How?

Reply:
next time don't commit it